docs/api-reference/core/widget.md

```diff
@@ -39,6 +39,14 @@ Additional inline CSS styles on the top HTML element of the widget. camelCase CS
 
 Additional CSS classnames on the top HTML element.
 
+#### `container` (string | HTMLDivElement, optional) {#container}
+
+The container that this widget is being attached to. Default to `viewId`.
+- If set to `'root'`, the widget is placed relative to the whole deck.gl canvas.
```

Comment:
Considering API options:
- Seems to me that `root` could also be a valid `placement`, which would then ignore viewId?

Reply:
Placement doesn't have any interactions with `viewId` in implementation, it just positions within a view (or root) box. Perhaps the name is a little confusing.. "alignment" would have been a great name since that values are like "top-left", except that "fill" is an option.

If we're open to a breaking change, "alignment" could be more clear to new users